src/app/external-log-in/email-confirmation/confirmation-sent/confirmation-sent.component.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+} from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+
+@Component({
+  selector: 'ds-confirmation-sent',
+  templateUrl: './confirmation-sent.component.html',
+  styleUrls: ['./confirmation-sent.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [TranslateModule],
+  standalone: true,
+
+})
+export class ConfirmationSentComponent { }
```

Comment:
While I realize this is a small component, all components should have a description in a comment or TypeDocs. It's just useful for other developers to more easily understand the purpose of each component